modules/bullet/SCsub

```diff
@@ -187,6 +187,7 @@ if env["builtin_bullet"]:
         "LinearMath/btGeometryUtil.cpp",
         "LinearMath/btPolarDecomposition.cpp",
         "LinearMath/btQuickprof.cpp",
+        "LinearMath/btReducedVector.cpp",
```

Comment:
It seems like you had omitted this `.cpp` file when updating to 2.90/master a while ago: 3e7db60d56d5c25d7aa3fded4b90f36ca341159c.

It built and seemed to work fine without it so maybe it's not needed, it seems to be used in:
```
BulletSoftBody/btDeformableContactProjection.cpp
190:    btReducedVector p(x.size());
```
But in the `#else` of an `#ifndef USE_MGS` (i.e. only used if we pass `USE_MGS`).

I'm adding it for consistency's sake, it should be optimized out by the linker if we don't actually need it I suppose. There's probably a lot of Bullet components which we wouldn't need to build if we don't use them directly, but not sure how easy that would be to identify to slim down our `SCsub`.